powerline/bindings/i3/bar.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
Bindings file should be named `powerline…`: e.g. `powerline-bar.py`. Note that this script is most likely be moved elsewhere and “elsewhere” does not necessary contain `powerline` in directory name.